src/components/Facility/FacilityCover.tsx

```diff
@@ -0,0 +1,75 @@
+import { navigate } from "raviger";
+import { useEffect } from "react";
+
+import useAuthUser from "@/hooks/useAuthUser";
+
+import * as Notification from "@/Utils/Notifications";
+
+import { Cancel, Submit } from "../Common/ButtonV2";
+import Page from "../Common/Page";
+import { CoverImage } from "./CoverImage";
+
+interface FacilityProps {
+  facilityId?: string;
```

Comment:
why nullable? won't we always have facilityId?

```suggestion
  facilityId: string;
```